### ISSUE TITLE
Select either "heuristic" or "transformation" to describe allowed age `TermURL`s for within- and across-tool consistency

### ISSUE THREAD

**Issue by alyssadai** [neurobagel/bagelbids#68]

**Comment by alyssadai** [comment]
@surchs I think since we're planning to accept `"bg:int"` and `"bg:float"`, "heuristic" might make more sense to describe these rather than "transformation".

**Comment by surchs** [comment]
Yeah, good point @alyssadai. Are you proposing that we consistently refer to this stuff as a "heuristic" and get rid of the "transformation" name?

**Comment by alyssadai** [comment]
Yes, if you/Jonathan agree

**Comment by alyssadai** [comment]
After discussion, we will pick "transformation". The annotation tool, CLI, and docs should check that we are using this term consistently in user-facing text.
